tests/container_disk_test.go

```diff
@@ -201,16 +194,14 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting from custom image location", func() {
 		Context("with disk at /custom-disk/downloaded", func() {
 			It("[test_id:1466]should boot normally", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation), "#!/bin/bash\necho 'hello'\n")
```

Comment:
This needs to be reverted as the image is concrete for the test.

Reply:
Oy, thanks for helping out.

Reply:
I think you discovered "bug" of `libvmi` The `WithContainerImage` is using the hardcoded name for the disk and therefore you can only provide one `WithContainerImage` option per VMI.  Otherwise, the PR looks great!